docs/demos/create-br1-linux-bridge.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkState
+metadata:
+  name: node01
+spec:
+  desiredState:
+    interfaces:
+      - name: eth1
+        type: ethernet
+        state: up
+      - name: br1
+        type: linux-bridge
+        state: up
+        bridge:
+          options:
+            stp:
+              enabled: false
+          port:
+            - name: eth1
+              stp-hairpin-mode: false
+              stp-path-cost: 100
+              stp-priority: 32
+        interfaces:
+      nodeName: node1
```

Comment:
Do we still have this one in the implementation? We match NodeNetworkState and Node by their metadata.name, I don't think we need this one anymore.

Reply:
I will add an issue to remove it.